source/index.html.md

```diff
@@ -793,3 +793,369 @@ Account | Income Statement:Revenue:Product Revenue
+                    [ "20", "50" ],
+                    [ "100", "110" ],
+                    [ "10", "20" ],
+                    [ "22", "34" ]
+                ]
+            },
+            {
+                "id": "17",
+                "sheet_range": "B7:E10",
+                "rows": {
+                    "1": [ "65", "66", "67", "68" ]
+                },
+                "columns": {
+                    "10": [ "29", "29", "33", "33" ],
+                    "6": [ "7", "8", "7", "8" ]
+                },
+                "filters": {
+                    "4": [ "5" ]
+                },
+                "values": [
+                    [ "12", "34", "99", "54" ],
+                    [ "99", "43", "76", "65" ],
+                    [ "12", "13", "23", "54" ],
+                    [ "65", "56", "34", "32" ]
+                ]
+            }
+        ]
+    },
+    {
+        "modified_at": "2019-09-24T17:46:17.086755+00:00",
+        "created_by": {
+            "id": "1"
+        },
+        "id": "7",
+        "created_at": "2019-09-24T17:46:17.086632+00:00",
+        "name": "Yet Another",
+        "reports": [
+            {
+                "id": "20",
+                "sheet_range": "B1:C5",
+                "rows": {
+                    "10": [ "30", "31", "32", "29", "RANDOM TITLE" ]
+                },
+                "columns": {
+                    "6": [ "9", "8" ]
+                },
+                "filters": {
+                    "4": [ "5" ],
+                    "1": [ "65" ]
+                },
+                "values": [
+                    [ "1", "3" ],
+                    [ "20", "50" ],
+                    [ "100", "110" ],
+                    [ "10", "20" ],
+                    [ "22", "34" ]
+                ]
+            },
+            {
+                "id": "21",
+                "sheet_range": "B7:E10",
+                "rows": {
+                    "1": [ "65", "66", "67", "68" ]
+                },
+                "columns": {
+                    "10": [ "29", "29", "33", "33" ],
+                    "6": [ "7", "8", "7", "8" ]
+                },
+                "filters": {
+                    "4": [ "5" ]
+                },
+                "values": [
+                    [ "12", "34", "99", "54" ],
+                    [ "99", "43", "76", "65" ],
+                    [ "12", "13", "23", "54" ],
+                    [ "65", "56", "34", "32" ]
+                ]
+            }
+        ]
+    }
+]
+```
+
+This endpoint fetches all the Report Templates available to the currently authenticated user. This includes all Report Templates anyone at the user's company has created.
+
+Report Templates are made up of one or many reports, each with a defined `sheet_range` where the report should be placed on the sheet.
+
+For rows, columns, and filters in a report, the response is a dictionary in the format `{top level dimension id: [list of individual rows/columns]}`
```

Comment:
I'd move this and the line above it to that table sort of thing I mentioned in my other comment.